sdk/metric/aggregator/aggregatortest/test.go

```diff
@@ -65,7 +64,7 @@ func newProfiles() []Profile {
 }
 
 func NewAggregatorTest(mkind sdkapi.InstrumentKind, nkind number.Kind) *sdkapi.Descriptor {
-	desc := metrictest.NewDescriptor("test.name", mkind, nkind)
+	desc := sdkapi.NewDescriptor("test.name", mkind, nkind, "", "")
```

Comment:
This was done to avoid an import loop.  But it is used in a few places to create a Descriptor without supplying a description or unit.  I think we might consider creating a helper function for those cases.

Reply:
> I think we might consider creating a helper function in the sdkapi for those cases.

Also, for the record, the originally public `sdkapi` package contained this helper function. It was moved into `metrictest` to take it out of the public namespace because there were dozens of test-only locations relied on the 1-line API. Since `sdkapi` is now SDK-internal, makes sense to move the helper back.